web/src/server/api/routers/media.ts

```diff
@@ -0,0 +1,74 @@
+import { z } from "zod";
+
+import { env } from "@/src/env.mjs";
+import { getMediaStorageServiceClient } from "@/src/features/media/server/getMediaStorageClient";
+import {
+  createTRPCRouter,
+  protectedProjectProcedure,
+} from "@/src/server/api/trpc";
+import { logger } from "@langfuse/shared/src/server";
+import { TRPCError } from "@trpc/server";
+
+export const mediaRouter = createTRPCRouter({
+  get: protectedProjectProcedure
```

Comment:
Is the trpc router case sensitive? Otherwise, GET may be preferable to be consistent with other records.

Reply:
This file sneaked in, was not ready yet to be part of this PR :)